domain.yml

```diff
@@ -201,6 +202,15 @@ templates:
       contributors 🚀.
   utter_ask_faq_languages:
   - text: You can use Rasa to build assistants in any language you want!
+  utter_ask_faq_is_programming_required:
+  - text: "Rasa is written in python, but programming is not required to develop an assistant. \n
+      To learn how to create an assistant, check out the [Rasa Masterclass](https://www.youtube.com/watch?v=rlAQWbhwqLA). \n\n
+      There is one place where programming skills come in handy, and that is with
+      [Custom Actions](https://rasa.com/docs/rasa/core/actions/#custom-actions). \n\n
+      You can create an action server in node.js, .NET, java, or any other language and
+      define your actions there - but we provide a small python SDK to make development there even easier. \n
+      For custom actions written in python see the [Rasa SDK](https://rasa.com/docs/rasa/api/rasa-sdk/#rasa-sdk)
```

Comment:
Capitalize: `Node.js`, `Java`, `Python` and L206

Reply:
(if we decide to use entities, we should use synonyms for that so they appear correctly in two stage fallback)

Reply:
I capitalized them here and elsewhere in domain.yml

Am going to introduce entities as part of another update. See comment above.